src/component/component.ts

```diff
@@ -165,6 +168,7 @@ export class Component<T extends Env, Props extends {}> {
       });
       depth = 0;
     }
+    this._triggerHook = this.env[portalSymbol as any] || null;
```

Comment:
I think that this should not be registered on the component.

First, it pollutes the name space (also, we have the convention of using two `_` for owl internal stuff in `Component`). Second, it is some additional work at the creation of a component (which is much more common than emitting events). So, in my opinion, the check should be done in the `trigger` method, every time an event is emitted (it is fine to do it each time, it's a very very small amount of work.

Reply:
:+1: